src/Runner.js

```diff
@@ -172,16 +161,63 @@ function run(transformFile, paths, options) {
         reject(e.message);
       });
     });
-  } else if (!fs.existsSync(transformFile)) {
-    process.stderr.write(
-      colors.white.bgRed('ERROR') + ' Transform file ' + transformFile + ' does not exist \n'
-    );
-    return;
-  } else {
-    return transform(transformFile);
+  } else if (fs.existsSync(transform)) {
+    let stats = fs.lstatSync(transform);
+
+    if (stats && stats.isFile(transform)) {
```

Comment:
I believe `lstatSync` always returns something, so testing for the truthiness of `stats` is unnecessary. [`isFile` doesn't accept an argument](https://nodejs.org/api/fs.html#fs_class_fs_stats).